components/Modals/Login.tsx

```diff
@@ -82,6 +82,8 @@ export default function LoginModal({ onClose, onLogin, open }) {
       isOpen={open}
       className="fixed inset-0 top-1/2 left-1/2 h-full w-screen -translate-x-1/2 -translate-y-1/2 transform bg-back p-5  pt-10 font-nunito text-white shadow-lg outline-none sm:absolute sm:h-fit sm:w-[550px] sm:rounded-lg sm:pt-5"
       contentLabel="Login Modal"
+      styles={{overlay:{
+      backgroundColor:'rgba(1,1,1,0.75)'}}
```

Comment:
It looks like you may have misunderstood, right now the modal is set to have a bg of 75% white I believe the request in #177 was to change it t be 50% black.

this particular code isn't compiling because it needs to be
```js
style={{
  overlay: {
  backgroundColor:'rgba(1, 1, 1, 0.75)'}
}}
```

Additionally since we prefer to use [tailwind](https://tailwindcss.com/) lets go ahead and try to describe this bg as a tailwind class
in this case you could use a one-off like `bg-[#00000080]` or `bg-[rgba(0,0,0,0.5)]`

Reply:
in this case you will want to use `overlayClassName` in addition to the standard `className`
maybe something like `"bg-[#01010180] fixed inset-0"`